core/roboconf-messaging-rabbitmq/src/main/java/net/roboconf/messaging/rabbitmq/RabbitMqClientDm.java

```diff
@@ -254,7 +268,7 @@ public void sendMessageToTheDm( Message msg ) throws IOException {
 	 * #listenToTheDm(net.roboconf.messaging.client.IClient.ListenerCommand)
 	 */
 	@Override
-	public void listenToTheDm( ListenerCommand command )
+	public synchronized void listenToTheDm( ListenerCommand command )
```

Comment:
Wow! Synchronized everywhere.
Please, add an issue to review all the synchronization in the messaging in our next iteration.